roles/os_hardening/README.md

```diff
@@ -280,6 +280,21 @@ We know that this is the case on Raspberry Pi.
 - `os_sha_crypt_max_rounds`
   - Default: `640000`
   - Description: Define the number of maximum SHA rounds. With a lot of rounds brute forcing the password is more difficult. But note also that it more CPU resources will be needed to authenticate users. The values must be inside the 1000-999999999 range.
+- `tmp_dir_mode`
+  - Default: '1777'
+  - Description: Define directory permissions of /tmp.
+- `tmp_mnt_enabled`
+  - Default: 'false'
+  - Description: Set to true to mount local disks to /tmp. You have to configure variable tmp_mnt_src to work.
```

Comment:
```suggestion
  - Description: Set to true to mount /tmp as a separate filesystem. You have to configure variable `tmp_mnt_src` or the mount will fail.
```

I have a slight issue with using "local disk" here, Mount manages filesystems and especially `/tmp` could also be a ramdisk.